doc/adoc/user_guide.adoc

```diff
@@ -0,0 +1,117 @@
+= SUSE Distribution Migration System
+Marcus Schäfer; Jesús Velázquez
```

Comment:
@fsundermeyer thank you for the feedback and suggestions.
Regarding the name, yes it is "Distribution Migration System" and it should remain like that.

Reply:
Yes that name was already send to the business people, so let's keep it

Reply:
Sigh, I already assumed that the name is fixed. How about at least using upgrade (the verb) in the text (rather than migrate)?

Reply:
@fsundermeyer, in the text., upgrade instead of migrate/migration or migrate ?

Reply:
Yes. Not when the product name is referred to, but 

To start ~migrating/the migration~ run ...
To start upgrading/the upgrade run ...

Reply:
Got it. I was applying it in that way.
I will apply the suggestions that were missed because they were hidden due to outdate, as well.

Reply:
@fsundermeyer, I fixed this change and the other remaining suggestions.
Please, if you could give an extra look in case I missed something or there is a part that is still not clear, it would be great.